layouts/index.html

```diff
@@ -24,74 +24,39 @@ <h1>{{ .Title}}</h1>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col"></div>
-
-			<figure>
-				<img class="screenshot" src="/img/les-meves-ajudes.png" alt="Les mes ajudes">
-				<figcaption><a href="https://ajuntament.barcelona.cat/lesmevesajudes/">Les meves ajudes</a> {{ $.Param "usecases.les_meves_ajudes.caption_1" }} <a href="https://openfisca.org/doc/openfisca-web-api/index.html">Web API</a> {{ $.Param "usecases.les_meves_ajudes.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.leximpact.title" }}</h3>
-				<p>{{ $.Param "usecases.leximpact.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/leximpact.png" alt="LexImpact">
-				<figcaption><a href="https://leximpact.beta.gouv.fr/">LexImpact</a> {{ $.Param "usecases.leximpact.caption_1" }} <a href="https://openfisca.org/doc/openfisca-python-api/index.html">Python API</a> {{ $.Param "usecases.leximpact.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.reforms.title" }}</h3>
-				<p>{{ $.Param "usecases.reforms.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/revenu-de-base.jpg" alt="Comment appréhender le coût d’un revenu de base ?">
-				<figcaption><a href="http://www.revenudebase.info/2017/04/07/apprehender-cout-dun-revenu-de-base/">Publication</a> {{ $.Param "usecases.reforms.caption_1" }} <a href="https://openfisca.org/doc/openfisca-python-api/index.html">Python API</a> {{ $.Param "usecases.reforms.caption_2" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col">
-				<h3>{{ $.Param "usecases.rapu_ture.title" }}</h3>
-				<p>{{ $.Param "usecases.rapu_ture.description" }}</p>
-			</div>
-
-			<figure>
-				<img class="screenshot" src="/img/rapu-ture.png" alt="Rapu Ture - Exploring the Rules">
-				<figcaption><a href="http://nz.openfisca.org/">Rapu Ture</a> {{ $.Param "usecases.rapu_ture.caption" }}</figcaption>
-			</figure>
-		</section>
-
-		<section class="row">
-			<div class="left-col"></div>
-
-			<figure>
-				<img class="screenshot" src="/img/legexp-nsw.png" alt="New South Wales's Legislation Explorer">
-				<figcaption><a href="http://nsw-rules.herokuapp.com/">New South Wales</a> {{ $.Param "usecases.legexp_nsw.caption_1" }} <a href="https://github.com/openfisca/legislation-explorer">Legislation Explorer</a> {{ $.Param "usecases.legexp_nsw.caption_2" }}</figcaption>
-			</figure>
-		</section>
+		{{ range $slug := .Param "homepage_featured" }}
+			{{ $showcase := index $.Site.Data.showcase $slug }}
+			<section class="row">
+					<div class="left-col">
+						{{ if eq $slug "mes_aides" | or $slug "leximpact" | or $slug "rapu_ture"}}
```

Comment:
I don't really understand your comment, it would be easier for me to discuss it.

As I remember, we had discussed not using a categories because reuses often belong to several categories, instead we have prototyped  the using of tags, which makes me see that I have totally forgotten them in this PR. Do you want me to implement this metadata `tags` ? 

If yes i can suggest two different solutions where `tags` reuses metadata 
-  contains the values in French and English (wordy but easy for the contribution)

```yml
tags:
  fr: [sondage, python, gouvernement]
  en: [survey, python, governement]
```

- contains `tag` ID (allows better control of the list of tags but it is less easy to contribute because you have to refer to a data file that lists the available tags and related identifiers)

```yml
tags: [2, 3, 7]
```
```yml
tags:
  - id: 2
    fr: gouvernement
    en: governement
  - id: 3
    fr: sondage
    en: survey
  - id: 7
    fr: python
    en: python
```

Reply:
No, you are right. This is an optimisation for the current layout but irrelevant for the future. Forget what I said 🙂 Sorry for the noise!